sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/SubstituteUnresolvedOrdinals.scala

```diff
@@ -27,11 +27,19 @@ import org.apache.spark.sql.types.IntegerType
  * Replaces ordinal in 'order by' or 'group by' with UnresolvedOrdinal expression.
```

Comment:
Update the doc of `SubstituteUnresolvedOrdinals`.

Reply:
Update. later since we need https://github.com/apache/spark/pull/30212 done